enterprise/frontend/src/embedding-sdk-package/package.template.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metabase/embedding-sdk-react",
-  "version": "0.57.0-alpha.3",
+  "version": "0.58.0-beta",
```

Comment:
FYI, in 57 I saw that beta also has `beta.0` and `beta.1`, but now that we have hosted bundles I guess we only need to publish a single beta version. 

<img width="1548" height="246" alt="CleanShot 2568-12-17 at 16 17 51@2x" src="https://github.com/user-attachments/assets/67c1efa0-11cc-427e-9461-5cb5fea84303" />

Reply:
I think realistically, we probably only need one beta. But in theory, there could be more if we make changes to sdk-package like we introduce bugs and fixes them for example.